src/module/nft/bpf.rs

```diff
@@ -0,0 +1,22 @@
+/// Global configuration passed down the BPF part.
+#[repr(C, packed)]
+pub(super) struct NftConfig {
+    /// Bitfield of events to collect based on the verdict.
+    /// The values don't follow the kernel definitions in order to
+    /// avoid to follow kernel internal changes.
```

Comment:
this is leftover. It was updated, but not included here (no refresh was done with this small description change).
Added a comment to keep track of it.